packages/wonder-blocks-core/components/clickable-behavior.test.js

```diff
@@ -79,22 +79,6 @@ describe("ClickableBehavior", () => {
         expect(button.state("pressed")).toEqual(false);
     });
 
-    it("changes focused state on tab key down/up", () => {
-        const onClick = jest.fn();
-        const button = shallow(
-            <ClickableBehavior disabled={false} onClick={(e) => onClick(e)}>
-                {(state, handlers) => {
-                    return <button {...handlers}>Label</button>;
-                }}
-            </ClickableBehavior>,
-        );
-        expect(button.state("focused")).toEqual(false);
-        button.simulate("keyup", {keyCode: keyCodes.tab});
-        expect(button.state("focused")).toEqual(true);
-        button.simulate("keydown", {keyCode: keyCodes.tab});
-        expect(button.state("focused")).toEqual(false);
-    });
-
```

Comment:
This test is now obsolete because I removed the tab-related code from `handleKeyUp` and `handleKeyDown`. The tab-focus behavior is now handled by `handleBlur` and `handleFocus`.